pkgs/by-name/dp/dprint/package.nix

```diff
@@ -56,6 +57,7 @@ rustPlatform.buildRustPackage rec {
     };
 
     updateScript = nix-update-script { };
+    plugins = dprint-plugins;
```

Comment:
https://github.com/NixOS/nixpkgs/pull/369415/files#r1899975891

Sorry, I don't understand how this `dprint.passthru.plugins` will work. :bow: 
When we inject the CLI in the plugin's check phase, does this not create a circular dependency?

If adding this does not check the schema compatibility, I suggest removing it until it can actually be used.

Reply:
it is an alias for `dprint-plugins`, but I removed it for now.

I am fine with having just a `dprint-plugins`, it is not really needed to make `dprint.plugins` work tbh.